### PR TITLE
T14102

### PR DESCRIPTION
**Pull request by joaquimrocha**
https://phabricator.endlessm.com/T14102

### CODE REVIEW

File: src/plugins/eos-external-apps-build-install [hunk 1]
```diff
@@ -121,6 +121,11 @@ else
     exit 1
 fi
 
+# Fix permissions on export dir because some debs' tarballs base directory
+# has read permissions only for the user, and thus wouldn't be usable as a
+# Flatpak runtime
+chmod a+rx "$EXPORTDIR"
```
Comment (cosimoc) [inline src/plugins/eos-external-apps-build-install]: Nitpick: you could omit `a`. But do you really also need execution permissions, in addition to read?

File: src/plugins/eos-external-apps-build-install [hunk 2]
```diff
@@ -82,10 +82,11 @@ EXTAPPSBUILDDIR="$HOME/external-apps-build"
 APPDIR="$EXTAPPSBUILDDIR/$APPNAME"
 BUILDDIR="$APPDIR/build"
 BUNDLEDIR="$APPDIR/bundle"
+EXPORTDIR="$BUNDLEDIR/usr"
```
Comment (cosimoc) [inline src/plugins/eos-external-apps-build-install]: Looks like this belongs to the next commit?

File: src/plugins/eos-external-apps-build-install [hunk 3]
```diff
@@ -82,10 +82,11 @@ EXTAPPSBUILDDIR="$HOME/external-apps-build"
...
 
 # Remove any previous left build dir and create it again
 rm -rf "$BUNDLEDIR"
-mkdir -p "$BUILDDIR" "$BUNDLEDIR"
+mkdir -p "$BUILDDIR"
```
Comment (cosimoc) [inline src/plugins/eos-external-apps-build-install]: Not sure this agrees with the commit message... You mention it was being called with the build directory twice, but it was actually called with both the build and bundle directories.

File: src/plugins/eos-external-apps-build-install (hunk 1 above)
Comment (joaquimrocha) [inline src/plugins/eos-external-apps-build-install]: The execution bit on a directory is for listing and we need it.

**Pull request merged**